testdata/csharp/csharp_10-dotnet_6/Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");

```

Comment:
Let's go with a meaningful message. At minimum print the name of the example (i.e. "C# 10 using .Net 6"). Ideally you can do something closer to the `go` examples where you check the version that is running using some library and error if it's an unexpected version.

Reply:
Added version checks, which error when the version is unexpected.